bin/ci

```diff
@@ -0,0 +1,173 @@
+# only)
+#
+# usage: install_compose [RELEASE]
+#
+# RELEASE : docker-compose release to install
+function install_compose() {
+    DOCKER_COMPOSE_VERSION=${1-1.18.0}
+
+    echo "Installing docker-compose (v$DOCKER_COMPOSE_VERSION) ..."
+
+    sudo rm /usr/local/bin/docker-compose
+    curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+    chmod +x docker-compose
+    sudo mv docker-compose /usr/local/bin
+}
+
+
+# install_dpendencies: build lms & fonzie containers, fetch node dependencies
+#
+# usage: install_dependencies
+#
+# nota bene: we do not set a custom user/group ID to run the node container,
+# doing so leads to permission issues in the Travis context.
+function install_dependencies() {
+    echo "Building fonzie containers..."
+
+    build lms
+    build fonzie
+    FONZIE_UNSET_USER=1 run --no-deps node yarn install --no-progress
+}
+
+
+# run_test_suite: run the test suite
+#
+# usage: run_test_suite
+function run_test_suite() {
+    echo "Starting the test suite..."
+
+    if [ -z $TOXENV ]
+    then
+        echo "Running API spec test suite..."
+
+        docker_compose --no-ansi up -d lms
+
+        # Wait for mysql database to accept connections
+        run fonzie dockerize -wait tcp://mysql:3306 -timeout 60s
+
+        local sql="$HOME/.cache/db/$OPENEDX_RELEASE.sql"
+        local database="edxapp"
+
+        if [ -e $sql ]
+        then
+            echo "Stack status:"
+            docker_compose ps
+
+            echo "MySQL container logs:"
+            docker_compose logs mysql
+
+            echo "Loading database schema from cache: $sql"
+            # FIXME: docker-compose is having issues with huge files streamed
+            # via a pty/stdin, so we bypass compose to use raw docker exec
+            # command.
+            docker exec -i fonzie_mysql_1 mysql -u root $database < $sql
+        else
+            # Perform database migrations
+            echo "Starting database migration"
+            run lms python ./manage.py lms migrate
+
+            # Dump database
+            echo "Dump database schema to: $sql"
+            exec_ mysql mysqldump -u root $database > $sql
+        fi
+
+        # Run API implementation tests
+        run node yarn dredd
+    else
+        echo "Running tox:$TOXENV test suite..."
+
+        run -e TOXENV fonzie tox
+    fi
+}
+
+
+# coverage_report: report test coverage
+#
+# usage: coverage_report
+function coverage_report() {
+    echo "Starting coverage report..."
+
+    run -e CODECOV_TOKEN fonzie codecov
+}
+
+
+function main() {
+    declare program="$0"
+
+    local install=0
+    local before_install=0
+    local script=0
+    local after_success=0
+
+    while [[ "$#" -ge "1" ]]
+    do
+        local key="$1"
+        shift
+
+        case "$key" in
+            --after-success)
+                after_success=1
+                ;;
```

Comment:
`after_success=1 ;;` ?

Reply:
:+1:

Reply:
I think it works but I am not sure it's widely used. I often read the original syntax.